Documentation/contributor-guide/triage_issues.md

```diff
@@ -4,43 +4,107 @@
+
+## Step 4 - Define the areas impacted
 
-## Validate if an issue is a bug
+Adding an `area` label to an issue helps create visibility on which areas of the etcd project require attention and helps contributors find issues to work on relating to their particular skills or knowledge of the etcd codebase.
 
-Validate if the issue is indeed a bug. If not, add a comment with findings and close trivial issue. For non-trivial issue, wait to hear back from issue reporter and see if there is any objection. If issue reporter does not reply in 30 days, close the issue. If the problem can not be reproduced or require more information, leave a comment for the issue reporter.
+If an issue crosses multiple domains please add additional `area` labels to reflect that.
 
-## Inactive issues
+Below is a brief summary of the area labels in use by the etcd project along with any comments on their use:
 
-Issues that lack enough information from the issue reporter should be closed if issue reporter do not provide information in 60 days.
+| Label                   | Comments                                                      |
+| ---                     | ---                                                           |
+| area/community          |                                                               |
+| area/raft               |                                                               |
+| area/clientv3           |                                                               |
+| area/performance        |                                                               |
+| area/security           |                                                               |
+| area/tls                |                                                               |
+| area/auth               |                                                               |
+| area/etcdctl            |                                                               |
+| area/etcdutl            |                                                               |
+| area/contrib            | Not to be confused with `area/community` this label is specifically used for issues relating to community maintained scripts or files in the `contrib/` directory which aren't part of the core etcd project. |
+| area/documentation      |                                                               |
+| area/tooling            | Generally used in relation to the third party / external utilities or tools that are used in various stages of the etcd build, test or release process, for example tooling to create sboms.          |
+| area/testing            |                                                               |
+| area/functional-testing |                                                               |
```

Comment:
It seems like it's been migrated to robustness testing.

Reply:
Good spotting - removed :)